src/NzbDrone.Core/IndexerSearch/Definitions/AlbumSearchCriteria.cs

```diff
@@ -7,12 +8,14 @@ public class AlbumSearchCriteria : SearchCriteriaBase
 
         public string AlbumTitle { get; set; }
         public int AlbumYear { get; set; }
+        public string Disambiguation { get; set; }
 
         public string AlbumQuery => GetQueryTitle(AlbumTitle);
+        public string DisambiguationQuery => GetQueryTitle(Disambiguation ?? string.Empty);
```

Comment:
Not sure we need another variable here, should we just include in album query if it’s not empty? Given it’s part of the album.

Reply:
yeah, good idea --> my only concern: is this giving good results on "SupportsAudioSearch" indexers? if yes --> much less work :-)

Reply:
Good point, something we should probably test